test-data/unit/check-flags.test

```diff
@@ -194,7 +194,7 @@ x = 0  # type: NoReturn  # E: Incompatible types in assignment (expression has t
 [builtins fixtures/dict.pyi]
 
 [case testNoReturnImportFromTyping]
-from typing import NoReturn
```

Comment:
This is not right. If it fails, then typeshed should be updated, this is why this test is called ``testNoReturnImportFromTyping``.